river/command/layout.zig

```diff
@@ -76,6 +76,7 @@ pub fn sendLayoutCmd(
         if (mem.eql(u8, layout.namespace, target_namespace)) break layout;
     } else return;
 
+    layout.layout.sendUserCommandTags(output.pending.tags);
```

Comment:
We need to check the version of the layout object here before sending the event. Otherwise, we will send it to clients that have bound the interface at version 1 and cause them to be terminated.